uber/configspec.ini

```diff
@@ -7,6 +7,10 @@
 # after a deadline or on-site.
 dev_box = boolean(default=True)
 
+# Redirect 404s to Uber's default URL
+default_url = string(default="/uber/common/")
+default_url_priority = integer(default=1)
```

Comment:
Ah, good catch. `path` and `default_url` do different things, so we still need `default_url` (it's a actually a sideboard setting that plugins can override). But `default_url` should really use the interpolated value of `path`.